docs/source/getting-started/content/proc_basic-troubleshooting.adoc

```diff
@@ -1,6 +1,20 @@
 [id="basic-troubleshooting_{context}"]
 = Basic troubleshooting
 
+[NOTE]
+====
+Each released `crc` binary expires 30 days after the release due to certificates embedded in the Openshift cluster.
+Using an older `crc` binary will result in an expired certificates error.
+
+To resolve this error:
+
+* link:https://github.com/code-ready/crc/releases[Download the latest CodeReady Containers release] and place the `crc` binary in your `$PATH`.
+* Remove the cluster with certificate errors using the `crc delete` command.
+* Set up the new release with the `crc setup` command.
+* Start the new cluster with the `crc start` command.
+
+====
+
```

Comment:
Not sure whether this should be a "NOTE", or its own bullet point.

Reply:
note would make more sense as it stands out.

Reply:
I'd say that a note admonition makes sense to highlight the data loss aspect, but having all of this information in an admonition block is not necessary. In fact, I think it would be better as its own `proc_<name>.adoc` procedure file which we can add to the `assembly_troubleshooting.adoc` file.

@praveenkumar, do you mind if I send you a PR to this branch to show you what I mean? I feel like that would be easier than typing it all out. :)

Reply:
@robin-owen please go ahead, I will close this PR in favor of yours :)